crates/nu-cli/src/completion/matchers/case_insensitive.rs

```diff
@@ -0,0 +1,41 @@
+use crate::completion::matchers;
+pub struct Matcher;
+
+impl matchers::Matcher for Matcher {
+    fn matches(&self, partial: &str, from: &str) -> bool {
+        from.to_ascii_lowercase()
+            .starts_with(partial.to_ascii_lowercase().as_str())
+    }
+}
+
+#[cfg(test)]
```

Comment:
nit: this one isn't really necessary. Technically, it's only getting applied to the first test function. It's pretty common to use this pattern though:

```rust
#[cfg(test)]
mod tests {
  use super::*;

  #[test]
  fn my_test { ... }
}
```

It's nice because it creates a nice boundary between test code and regular code.

Feel free to use that style here and elsewhere. IF you'd rather keep it as is, which is fine too, you can drop `#[cfg(test)]`.

(note this comment applies to some other files too).

Reply:
ah, OK. makes total sense.

Fixing now.